features/GEM052_Correct-geometric-subcontexts.feature

```diff
@@ -24,8 +24,8 @@ Reference: https://github.com/buildingSMART/Sample-Test-Files/issues/137.
 
         Examples: 
             | schema | source |
-            | IFC4.3 | valid_RepresentationIdentifier_IFC4.3.csv |
```

Comment:
In this case we can just remove the outline. and start with the same given as above:

Given a model with Schema "IFC4.3" or "IFC4"

Reply:
That's true. I think we can even remove the schema selection from the Given statements; they are already selected in the python implementation. So then, we'll simply have three scenario's; 
- Shape representation identifiers
- Shape representation types
- Topology representation types

Something like [this](https://github.com/buildingSMART/ifc-gherkin-rules/pull/319/commits/ceee0cb293a88dd666d96a1fbad3ee4d5beeb37c)?